Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -49,6 +58,61 @@ - (BOOL)managedObjectContext:(NSManagedObjectContext *)context
     return result;
 }
 
+- (NSString *)descriptionForOperations:
+                  (NSDictionary<NSString *, NSDictionary<NSString *, NSNumber *> *> *)operations
+                             inContext:(NSManagedObjectContext *)context
+{
+    NSMutableArray *resultParts = [NSMutableArray new];
+
+    NSDictionary<NSString *, NSNumber *> *inserts = operations[@"INSERTED"];
+    if (inserts.count > 0) {
+        if (inserts.count == 1) {
+            [resultParts addObject:[NSString stringWithFormat:@"INSERTED %@ '%@'",
+                                             inserts.allValues[0], inserts.allKeys[0]]];
+        } else {
+            [resultParts addObject:[NSString stringWithFormat:@"INSERTED %lu itens",
+                                             (unsigned long)context.insertedObjects.count]];
+        }
+    }
+
+    NSDictionary<NSString *, NSNumber *> *updates = operations[@"UPDATED"];
+    if (updates.count > 0) {
+        if (updates.count == 1) {
+            [resultParts addObject:[NSString stringWithFormat:@"UPDATED %@ '%@'",
+                                             updates.allValues[0], updates.allKeys[0]]];
+        } else {
+            [resultParts addObject:[NSString stringWithFormat:@"UPDATED %lu itens",
+                                             (unsigned long)context.updatedObjects.count]];
+        }
+    }
```

Comment:
`m`: We have almost the same code block three times. Can we maybe extract a function here?